README.md.handlebars

```diff
@@ -230,6 +232,12 @@ docker-compose.yaml, it will ignore an invalid GPG key during the installation.
 
 *Please note though*, that this affects the security of the system and should not be used constantly.
 
+### When i try to start cloud control, it keeps exiting with "./ccc: exit status 1" and i want to debug that more
+
+Use the `docker logs` command with the failed container to see the complete log output. You can enhance the log by using the
+"DEBUG_<feature>" options. If you are really stuck, you can convince the container to keep running by setting "CONTINUE_ON_ERROR=yes" as an
+environment variable in the docker-compose file. Then you can debug with the running container.
```

Comment:
Do we need to add something like "using docker exec ..." or should it be clear to people?

Reply:
As said before, i would not explain it too far.
Or we describe it REALLY specific for a noob...
But an admin, who uses cloud control, should know how to handle containers i would say...

Reply:
Allright